### PR TITLE
Fixes #1169: Topic-revision - Tablet (Portrait) (Highfi)

### LINKED ISSUE #1169: Topic-revision - Tablet (Portrait) (Highfi)
Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/0e461d40-e024-4d56-9ae5-9697b1bbf450/P-Topic-Page-Revision-Tab

Implement high-fi UI for **Topic-revision** tablet portrait mode

**Target PR date**: 13 July 2020
**Target completion date**: 17 July 2020

### PR DESCRIPTION
**Pull request by MohamedMedhat1998**
## Explanation

- Fixes #1169 
- Implemented high-fi UI for **Topic-Revision** for portrait mode - tablet
- Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/0e461d40-e024-4d56-9ae5-9697b1bbf450/P-Topic-Page-Revision-Tab
### MDPI Emulator (1024x768)
screenshot-2020-07-13_16 56 44 637

screenshot-2020-07-13_16 56 48 58

### XHDPI Emulator (2560x1800)
screenshot-2020-07-13_16 53 12 646

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -154,4 +154,4 @@
       android:name=".walkthrough.WalkthroughActivity"
       android:theme="@style/OppiaThemeWithoutActionBar" />
   </application>
-</manifest>
+</manifest>
```
Comment (rt4914) [inline app/src/main/AndroidManifest.xml]: EOF is missing, add one blank line at the end.

File: app/src/main/res/layout/activity_topic_revision_test.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (rt4914) [inline app/src/main/res/layout/activity_topic_revision_test.xml]: Remove this file.

**Review by rt4914 — CHANGES_REQUESTED** [review]
LGTM - nit changes.

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (MohamedMedhat1998) [reply app/src/main/AndroidManifest.xml]: Done

File: app/src/main/res/layout/activity_topic_revision_test.xml (hunk 2 above)
Comment (MohamedMedhat1998) [reply app/src/main/res/layout/activity_topic_revision_test.xml]: Done

**Comment by MohamedMedhat1998** [comment]
@rt4914 merging is blocked, can you approve the changes, please?

**Review by rt4914 — APPROVED** [review]
LGTM

**Pull request merged**